### ISSUE TITLE
XDG_STATE_HOME

### ISSUE THREAD

**Issue by YodaEmbedding** [jhofscheier/ltex-utils.nvim#4]
The cache can be removed at any time. (e.g. Every 1 hour.)

It's better to store long-term user state in `XDG_STATE_HOME`:

```lua
vim.api.nvim_call_function("stdpath", {"state"}) .. "/ltex/"
```

**Comment by jhofscheier** [comment]
Good point. I have updated the default configuration to the suggested user state path in `XDG_STATE_HOME`.

Thanks for carefully checking the plugin. I much appreciate it.
